ci-operator/step-registry/openshift-qe/evpn/openshift-qe-evpn-commands.sh

```
@@ -0,0 +1,141 @@
+oc patch networks.operator.openshift.io cluster --type=merge -p \
+  '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"gatewayConfig":{"routingViaHost":true,"ipForwarding":"Global"}}}}}'
+oc wait co/network --for=condition=Progressing=False --timeout=10m
+
+# 3. Enable FRR and Route Advertisements
+oc patch Network.operator.openshift.io cluster --type=merge -p='{"spec":{"additionalRoutingCapabilities":{"providers":["FRR"]},"defaultNetwork":{"ovnKubernetesConfig":{"routeAdvertisements":"Enabled"}}}}'
+oc wait co/network --for=condition=Progressing=False --timeout=10m
+
+# 4. Upgrade FRR image (scale CVO down, set network operator unmanaged)
+oc scale -n openshift-cluster-version deployment.apps/cluster-version-operator --replicas=0
+oc patch Network.operator.openshift.io cluster --type='merge' -p='{"spec":{"managementState":"Unmanaged"}}'
+oc set image daemonset/frr-k8s -n openshift-frr-k8s frr=${FRR_IMAGE} reloader=${FRR_IMAGE}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)